src/sentry/static/sentry/app/views/organizationIntegrations/integrationStatus.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import CircleIndicator from 'app/components/circleIndicator';
+import theme from 'app/utils/theme';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {withTheme} from 'emotion-theming';
+
+import {colors} from './constants';
+
+type Props = {
+  status?: string;
+  enabled?: boolean;
```

Comment:
@NisanthanNanthakumar Passing both `enabled` and `status` is confusing. We should just pass the `status` and let first-party integrations and plugins figure out the status based off whether they are enabled or not.